src/harness/rwcRunner.ts

```diff
@@ -65,6 +65,15 @@ module RWC {
                     opts.options.noEmitOnError = false;
                 });
 
+                if (!useCustomLibraryFile) {
+                    let defaultLibPath = ts.sys.resolvePath("built/local/lib.d.ts");
+                    let defaultLib = {
+                        unitName: ts.normalizePath(defaultLibPath),
+                        content: Harness.IO.readFile(defaultLibPath)

```

Comment:
we are reading this file for every test. the file did not change, we should cache it.